lib/screens/form_screen.dart

```diff
@@ -12,81 +14,86 @@ class FormScreen extends StatelessWidget {
   final FocusNode typeFocusNode = FocusNode();
   final FocusNode descriptionFocusNode = FocusNode();
 
-
   @override
   Widget build(BuildContext context) {
     var theme = Theme.of(context);
-    return GestureDetector(
-      onTap: () => FocusScope.of(context).requestFocus(FocusNode()),
-      child: Scaffold(
-        body: Container(
-          padding: EdgeInsets.symmetric(horizontal: SPACING_16),
-          child: SafeArea(
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.stretch,
-              children: <Widget>[
-                SizedBox(
-                  height: SPACING_32,
-                ),
-                Text(
-                  'Preencha os dados abaixo',
-                  style: theme.textTheme.headline5,
-                ),
-                SizedBox(
-                  height: SPACING_16,
-                ),
-                InputField(
-                  onChanged: (text) {},
-                  focusNode: nameFocusNode,
-                  labelText: 'Número da placa antiga',
-                  isValid: true,
-                ),
-                SizedBox(
-                  height: SPACING_16,
-                ),
-                InputField(
-                  onChanged: (text) {},
-                  focusNode: cnpjFocusNode,
-                  labelText: 'Nova placa',
-                  isValid: true,
-                ),
-                SizedBox(
-                  height: SPACING_16,
-                ),
-                InputField(
-                  onChanged: (text) {},
-                  focusNode: modelFocusNode,
-                  labelText: 'Modelo',
-                  isValid: true,
-                ),
-                SizedBox(
-                  height: SPACING_16,
-                ),
-                InputField(
-                  onChanged: (text) {},
-                  focusNode: typeFocusNode,
-                  labelText: 'Tipo',
-                  isValid: true,
-                ),
-                SizedBox(
-                  height: SPACING_16,
-                ),
-                InputField(
-                  onChanged: (text) {},
-                  focusNode: descriptionFocusNode,
-                  labelText: 'Descrição',
-                  isValid: true,
-                ),
-                SizedBox(
-                  height: SPACING_48,
-                ),
-                PrimaryButton(
-                  text: 'Continuar',
-                  onPressed: () {
-                    Navigator.pushNamed(context, '/form');
-                  },
-                ),
-              ],
+    return Form(
```

Comment:
Acho que a gente tem que colocar esse form dentro do body do Scaffold. O Scaffold é onde a gente pode colocar `body`, `appBar`, `bottomBar`. Se a gente for colocar uma appbar, o form n pode estar em volta de tudo